kafka-consumer/src/main/java/com/codingdecoded/sunchit/kafka/consumer/consumer/AnalyticsConsumerService.java

```diff
@@ -0,0 +1,36 @@
+package com.codingdecoded.sunchit.kafka.consumer.consumer;
+
+import com.codingdecoded.sunchit.kafka.consumer.model.DriverLocation;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AnalyticsConsumerService {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Please use the autowired instance if defined somewhere; otherwise, create a new bean for it so that it gets registered by Spring.